commands/members/add.js

```diff
@@ -3,46 +3,96 @@
 let cli = require('heroku-cli-util')
 let co = require('co')
 let extend = require('util')._extend
+let Utils = require('../../lib/utils')
 
 function * run (context, heroku) {
-  // Users receive `You'll be billed monthly for teams over 5 members.`
-  // message when going over the FREE_TEAM_LIMIT
-  const FREE_TEAM_LIMIT = 5
-  let org = context.org
+  let orgInfo = yield Utils.orgInfo(context, heroku)
+  let groupName = context.org || context.flags.team
+
+  let warnMembershipLimit = function (totalMembers) {
+    // Users receive `You'll be billed monthly for teams over 5 members.`
+    if (totalMembers === 6) {
```

Comment:
Ouch...is there a way to avoid hard-coding this? At the least have a constant somewhere in the code?